# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/wdiff/strstr.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Well - that's a new one. A program defines it's own prototype instead of
using the system headers.

 - Clifford

--- ./wdiff.c~	Sun Nov  6 08:57:23 1994
+++ ./wdiff.c	Thu Sep 24 20:20:46 1998
@@ -59,7 +59,7 @@
 # endif
 #endif
 
-char *strstr ();
+// char *strstr ();
 
 #if HAVE_TPUTS
 # if HAVE_TERMCAP_H
